clients/include/testing_set_get_atomics_mode.hpp

```diff
@@ -19,7 +19,7 @@ using namespace std;
 
```

Comment:
Same here and in set_get_vector and set_get_vector_async

Reply:
Well if they had few includes I didn't update.  I guess I can change them too, unless they won't get benched like atomics, unless maybe we should bench atomics.